lang/en.js

```diff
@@ -46,8 +47,8 @@ export default {
 	"components.input-time-range.startTime": "Start Time",
 	"components.list-item-drag-handle.default": "Reorder item action for {name}",
 	"components.list-item-drag-handle.keyboard": "Reorder item, current position {currentPosition} out of {size}. To move this item, press up or down arrows.",
-	"components.menu-item-return.return": "Returns to previous menu.",
-	"components.menu-item-return.returnCurrentlyShowing": "Returns to previous menu. You are viewing {menuTitle}.",
+	"components.menu-item-return.return": "Return to previous menu.",
+	"components.menu-item-return.returnCurrentlyShowing": "Return to previous menu. You are viewing {menuTitle}.",
```

Comment:
Jeff requested this update, which I think sounds better.  We were both on the fence about whether filter should reuse the `components.menu-item-return.returnCurrentlyShowing` langtterm or make its own.  Pros and cons to both, so if someone feels strongly either way let me know.